packages/vuetify/src/locale/af.ts

```diff
@@ -8,9 +8,12 @@ export default {
   dataTable: {
     itemsPerPageText: 'Rye per bladsy:',
     ariaLabel: {
-      sortDescending: ': Sorted descending. Activate to remove sorting.',
-      sortAscending: ': Sorted ascending. Activate to sort descending.',
-      sortNone: ': Not sorted. Activate to sort ascending.',
+      sortDescending: 'Sorted descending.',
```

Comment:
I know it's nit-picky but could these be in alpha order?

Reply:
I don't really feel like changing this in 35 files.